modules/ossm-remove-control-plane.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/removing-ossm.adoc
+
+[id="ossm-remove-control-plane_{context}"]
+= Removing the control plane
+
+Follow this procedure to remove the {ProductName} control plane.
+
+.Prerequisites
+
+* You must log in as a cluster administrator to run these commands.
+
+.Procedure
+
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the Operator to begin uninstalling everything it installed.
+====
+
+
+. Run this command to retrieve the name of the installed custom resource:
++
+
+----
+$ oc get servicemeshcontrolplanes -n istio-system
+----
```

Comment:
`smcp` is a short name for `servicemeshcontrolplanes`. So the command above could be simplified to `oc get smcp -n istio-system`. Is it worth to replace it with the shorter version or even mentioning both?

Reply:
I am going to add a note making people aware of it, but I'll use the full command in the procedures.